week3.js

```diff
@@ -0,0 +1,223 @@
+// String and Array challenges
+
+// 1 strings:
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/,/g, ' ');        // replace commas with spaces
+console.log(myString);
+
+
+
+// 2 Arrays:
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+favoriteAnimals.push("turtle");  // adding Mauro's favorite animal at the end of the previous array
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, "meerkat");
+console.log("using splice() a method that changes the contents of an array by removing or replacing existing elements and/or adding new elements)\n \
+in .splice(1, 0, 'meerkat') the 1 defines the index that i want to work with and 0 defines the number of elements that i want to replace.\n \
+so the value of the array will be:\n \
+[\"blowfish\", \"meerkat\", \"capricorn\", \"giraffe\", \"turtle\"]");
+console.log(favoriteAnimals);
+console.log("The array has a length of: " + favoriteAnimals.length);
+favoriteAnimals.splice(3,1);   // delete giraffe from the previous array
+console.log(favoriteAnimals);
+console.log("The item you are looking for is at index: " + favoriteAnimals.findIndex(animal => animal === "meerkat"));
+
+
+
+
+// More JavaScript
+
+//1
+function sum(x, y, z) {
+  const result = x + y + z;
+  return result;
```

Comment:
You can even return the statement and don't need to make a const; `return x + y + z`